contracts/Aavegotchi/libraries/LibGotchiLending.sol

```diff
@@ -215,7 +213,8 @@ library LibGotchiLending {
         address lender = lending.lender;
 
         if (lending.initialCost > 0) {
-            LibERC20.transferFrom(s.ghstContract, _borrower, lender, _initialCost);
+            (bool success, ) = payable(lender).call{value: _initialCost}("");
```

Comment:
perhaps we can move this unsafe call to the end of the function call?

Reply:
Any specific reason? Or you just think it's good practice? @Timidan

Reply:
good practice